config/postgres/initdb/scripts/create-tables-and-functions.sql

```diff
@@ -72,7 +72,7 @@ WITH USER_INSERT_RESULT AS
             INSERT INTO AUTH (auth_id, hashed_password, user_id)
             SELECT uuid_generate_v4(), $2, user_id FROM USER_INSERT_RESULT;
             
-SELECT USERS.USER_ID::varchar, EMAIL, ADMIN, AUTH_ID::varchar, HASHED_PASSWORD
+SELECT USERS.USER_ID::varchar, EMAIL, ADMIN, AUTH_ID::varchar, HASHED_PASSWORD, DATE_CREATED, DATE_MODIFIED
```

Comment:
Does using `UUID` type work? As in`SELECT USERS.USER_ID::uuid, ...`? If not, let's open a bug for it.

In the meantime let's be consistent (you use `::text` below in the test).

Reply:
I did start converting these to strings because converting the psycopg result to a dict gets uphappy about uuid type. Sounds good to look into this but for now I'll just make everything text type.